pwndbg/hexdump.py

```diff
@@ -12,40 +12,44 @@
 import pwndbg.color
 
 
+color_scheme = None
+printable = None
+
 def groupby(array, count, fill=None):
     array = copy.copy(array)
     while fill and len(array) % count:
         array.append(fill)
     for i in range(0, len(array), count):
         yield array[i:i+count]
 
-#
-# We want to colorize the hex characters
-#
-color_scheme = {i:pwndbg.color.normal("%02x" % i) for i in range(256)}
-
-for c in bytearray((string.ascii_letters + string.digits + string.punctuation).encode('utf-8', 'ignore')):
-    color_scheme[c] = pwndbg.color.bold("%02x" % c)
-
-for c in bytearray(b'\x00\xff'):
-    color_scheme[c] = pwndbg.color.red("%02x" % c)
-
-for c in bytearray(b'\xff\x7f\x80'):
-    color_scheme[c] = pwndbg.color.yellow("%02x" % c)
+def load_color_scheme():
```

Comment:
On further evaluation, the general idea of runtime setting changes *is* useful, so I'm not opposed for this to be the first use-case (and it is an easily-observable one).